packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1876,6 +1592,19 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 this.updateDocumentDirtyState(false);
             }
 
+            if (message.metadata?.compressed) {
+                this.logger.sendTelemetryEvent({
```

Comment:
We should not generate telemetry event per op - that would create a ton of traffic and would be extremely expensive.
Please log either aggregates or do sampling.